src/util/util.ts

```diff
@@ -65,7 +67,7 @@ export const delay = (ms) => new Promise((res) => setTimeout(res, ms));
 export const processNumber = (
   num: number,
   min: number,
-  max: number
+  max: number,
```

Comment:
```suggestion
  max: number
```
Dont add comma here.